### PR TITLE
SQL Server transport queue schema support updates

### PR DESCRIPTION
**Pull request by mauroservienti**
**DO-NOT-MERGE** until v.1.2.3 is released.

update SQL Server transport documents to reflect changes introduced in the Queue Schema feature back-ported to v1.2.3.

@SimonCropp can you please review? Can you also suggest if other documents need to be updated?

/cc @danielmarbach @adamralph 
related to https://github.com/Particular/FeatureDevelopment/issues/320

### CODE REVIEW

File: Snippets/Snippets_5/Transports/SqlServer/MultiDb.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-  <!--startcode sqlserver-multidb-current-endpoint-schema-config 2.1 -->
+  <!--startcode sqlserver-multidb-current-endpoint-schema-config 1.2.3 -->
```
Comment (SimonCropp) [inline Snippets/Snippets_5/Transports/SqlServer/MultiDb.config]: this is weird. 2.1 -> 1.2.3. is that correct?

Comment (mauroservienti) [inline Snippets/Snippets_5/Transports/SqlServer/MultiDb.config]: Yes, the feature will be backported to v1.2.3

**Comment by SimonCropp** [comment]
this looks good to me. merge when the release happens

**Comment by mauroservienti** [comment]
released https://github.com/Particular/NServiceBus.SqlServer/releases/tag/1.2.3
Merging

**Pull request merged**